application/views/survey/questions/answer/multiplenumeric/rows/dynamic_slider.twig

```diff
@@ -22,7 +22,7 @@
                 </div>
             {% endif %}
             <div id="remainingvalue_{{ id }}" class="form-control-static numeric dynamic-remaining" data-number="1"><!-- alternative class : form-control : display like an input:text -->
-                {{ '{'~sumRemainingEqn~'}' }}
+                {{ sumRemainingEqn }}
```

Comment:
No way to send XSS with XSS activated ?

Reply:
Can you ellaborate more or give an example?